renew.py

```diff
@@ -108,16 +130,17 @@ def exit_with_error(message):
             confirmed_hosts = 0
 
             for host in hosts:
+                current_host = host.find_element(by=By.TAG_NAME, value="a").text
+                print("Confirming \"" + current_host + "\"")
```

Comment:
Perhaps we should update this message to something like "Checking if host  needs confirmation". What do you think?